job/src/main/java/org/jenkinsci/plugins/workflow/job/AfterRestartTask.java

```diff
@@ -33,11 +33,13 @@
 import hudson.model.queue.CauseOfBlockage;
 import java.io.IOException;
 import org.acegisecurity.Authentication;
+import org.kohsuke.stapler.export.ExportedBean;
 
 /**
  * Represents a {@link WorkflowRun} still running after a Jenkins restart.
  * Could be a {@code ContinuedTask}, though not really necessary since it is a {@link FlyweightTask} which would never be blocked anyway.
  */
+@ExportedBean
 class AfterRestartTask extends AbstractQueueTask implements Queue.FlyweightTask, Queue.TransientTask {
 
     private final WorkflowRun run;
```

Comment:
This is what should be `@Exported` I think; this class is just a proxy for the run.